app/allocation/controllers/view.js

```diff
@@ -5,7 +5,7 @@ const presenters = require('../../../common/presenters')
 function viewAllocation(personEscortRecordFeature = false) {
   return (req, res) => {
     const { allocation, canAccess } = req
-    const { moves, status } = allocation
+    const { moves, status, totalSlots } = allocation
     const bannerStatuses = ['cancelled']
     const movesWithoutProfile = moves.filter(move => !move.profile)
```

Comment:
Look at what these two variables (`movesWithoutProfile` and `movesWithProfile`) are trying to establish, it looks remarkably like `filledSlots` and `unfilledSlots`...could we replace them with the counts now coming from the API so the frontend does less of that heavy lifting?

Reply:
Not yet. Because we don't have those values when we ask for a single allocation.... yet.

Reply:
@solidgoldpig If it's a separate improvement, maybe we should capture it on the backlog so we can keep track of it?